.github/workflows/release-connect-v9-poduction.yml

```diff
@@ -0,0 +1,71 @@
+name: "[Release] connect v9 production"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+
+jobs:
+  extract-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set-version.outputs.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Extract connect version
+        id: set-version
+        run: echo "version=$(node ./ci/scripts/get-connect-version.js)" >> $GITHUB_OUTPUT
+
+  # set the rollback
+  sync-rollback-connect-v9:
+    if: startsWith(github.ref, 'refs/heads/release/connect/')
+    environment: production
+    name: "Backing up current production version ${{ needs.extract-version.outputs.version }} to rollback bucket"
+    runs-on: ubuntu-latest
+    env:
+      LATEST_VERSION: 9
+    steps:
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::538326561891:role/gh_actions_trezor_suite_prod_deploy
+          aws-region: eu-central-1
+
+      - uses: actions/checkout@v4
+        run: |
+          aws s3 sync "s3://connect.trezor.io/${{ env.LATEST_VERSION }}/" "s3://rollback-connect.trezor.io/${{ env.LATEST_VERSION }}/"
+
+  # From staging move it to production
+  deploy-staging-v9:
+    if: startsWith(github.ref, 'refs/heads/release/connect/')
+    # We deploy to production only if rollback sync was successful.
+    needs: [sync-rollback-connect-v9]
+    environment: production
+    name: "Deploying to staging-connect.trezor.io/9/"
+    runs-on: ubuntu-latest
+    env:
+      CURRENT_VERSION: ${{ needs.extract-version.outputs.version }}
+      LATEST_VERSION: 9
+    steps:
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::538326561891:role/gh_actions_trezor_suite_prod_deploy
+          aws-region: eu-central-1
+
+      - run: |
```

Comment:
and here is no checkout, so i would remove it from the step in rollback. Also name the step what it does.

Reply:
Name like here https://github.com/trezor/trezor-suite/pull/12192/commits/ba951cebf16a76d8dd6d23fded1bfc5ab571e733 ?

Reply:
yeah this should be fine :)